src/Console/ModelsCommand.php

```diff
@@ -450,81 +450,73 @@ protected function getPropertiesFromMethods($model)
                 } elseif (!method_exists('Illuminate\Database\Eloquent\Model', $method)
                     && !Str::startsWith($method, 'get')
                 ) {
-                    //Use reflection to inspect the code, based on Illuminate/Support/SerializableClosure.php
                     $reflection = new \ReflectionMethod($model, $method);
 
-                    $file = new \SplFileObject($reflection->getFileName());
-                    $file->seek($reflection->getStartLine() - 1);
+                    if ($reflection->getNumberOfParameters()) {
+                        continue;
+                    }
 
-                    $code = '';
-                    while ($file->key() < $reflection->getEndLine()) {
-                        $code .= $file->current();
-                        $file->next();
+                    $returnType = $reflection->getReturnType();
+
+                    if ($returnType === null) {
+                        $relationObj = $model->$method();
```

Comment:
You might trigger some unwanted behaviour here. You are basically calling every method on the model. (Imagine you have a method which sends a mail or something.)